packages/suite-web/e2e/tests/onboarding/firmware-update.test.ts

```diff
@@ -63,7 +63,7 @@ describe.skip('fw update from empty device bootloader 2.0.3 to firmware 2.5.1',
         cy.prefixedVisit('/');
     });
 
-    it('firmware update error', () => {
+    it.skip('firmware update error', () => {
```

Comment:
why skip? i would rather have it fixed or removed is test is no longer valid

Reply:
We agreed with @mroz22 to skip it so that QA can fix it later :) The problem with this test is that it mocks something and the mock stopped working and I don't know how to fix it :(